source/05-EL/07.ptx

```diff
@@ -59,12 +59,12 @@
       <task>
         <statement>
           <p>
-            According to the function given, if Mr. Abacus leaves his coffee on his desk all day, what will the coffee eventually cool to?
+            According to the function given, if Mr. Abacus leaves his coffee on his desk all day, what will the coffee eventually cool to? (Hint: Think about what happens to the function as <m>x\to \infty</m>.)
             <ol marker= "A." cols="2">
-              <li><m>90</m></li>
-              <li><m>70</m></li>
-              <li><m>20</m></li>
-              <li><m>0</m></li></ol>
+              <li><m>90</m><degree/><m>C</m></li>
+              <li><m>70</m><degree/><m>C</m></li>
+              <li><m>20</m><degree/><m>C</m></li>
+              <li><m>0</m><degree/><m>C</m></li></ol>
           </p>
         </statement>
         <answer>
```

Comment:
```suggestion
<hint><p>Think about what happens to the function as <m>x\to \infty</m>.</p></hint>
        <answer>
```

Reply:
I think that looks better. I've made the changes.